Doc/library/pathlib.rst

```diff
@@ -541,13 +541,11 @@ Pure paths provide the following methods and properties:
    reserved under Windows, ``False`` otherwise.  With :class:`PurePosixPath`,
    ``False`` is always returned.
 
-      >>> PureWindowsPath('nul').is_reserved()
-      True
-      >>> PurePosixPath('nul').is_reserved()
-      False
+   .. versionchanged:: 3.13
+      Windows path names that contain a colon, or end with a dot or a space,
+      are considered reserved. UNC paths may be reserved.
 
-   File system calls on reserved paths can fail mysteriously or have
-   unintended effects.
+   .. deprecated-removed:: 3.13 3.15
```

Comment:
Not sure the best way to say "use `ntpath.isreserved` instead" is, but we should say it.